migrations/2021-06-21-093252_create_games/up.sql

```diff
@@ -0,0 +1,29 @@
+CREATE TABLE games (
+    id INTEGER PRIMARY KEY
+);
+
+-- Going to create a separate table for players to avoid player_1_id, player_2_id etc columns in future.
+CREATE TABLE players (
+    id INTEGER PRIMARY KEY,
+    game_id INTEGER NOT NULL,
+    user_id INTEGER NOT NULL,
+    won BOOLEAN, -- Can be set per player, so that in future, multiple players (e.g. in a team) can win.
```

Comment:
i think maybe this should be `position`, or `points` or something, since if we get into many-player games, we may have 1st 2nd 3rd 4th, or alternatively something like 3pts, 2pts, 1pt, 0pts etc.